### ISSUE TITLE
Object.require is deprecated.

### ISSUE THREAD

**Issue by esden** [gsmcmullin/atom-gdb-debugger#11]
Use `remote.Dialog` instead of `remote.require("dialog")`
```
Object.require (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/electron-shims.js:57:12)
Object.<anonymous> (/Users/esdentem/.atom/packages/atom-gdb-debugger/lib/config-view.coffee:12:19)
Object.<anonymous> (/Users/esdentem/.atom/packages/atom-gdb-debugger/lib/config-view.coffee:259:4)
Module._compile (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/native-compile-cache.js:103:30)
Object..coffee (/Applications/Atom Beta.app/Contents/Resources/app.asar/src/compile-cache.js:216:21)
Module.load (module.js:458:32)
```

**Comment by gsmcmullin** [comment]
I believe this is fixed by 3b9b2cf.  Please confirm.

**Comment by esden** [comment]
Tested with Atom 1.13 beta1 and it seems to be working fine. Also no deprication report about this any more.
